examples/eventlisteners/eventlistener-embedded-bindings.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: listener-embed-binding
+spec:
+  serviceAccountName: tekton-triggers-example-sa
+  triggers:
+    - name: foo-trig
+      bindings:
+        - name: pipeline-binding
+          ref: pipeline-binding
+        - name: message-binding
```

Comment:
Just an idea to consider (we don't have to do it in this PR) -- given that the bindings spec is relatively simple (just a list of name and values), I wonder if something like this would work:

```
 bindings:
        - ref: pipeline-binding
        - name: message
          value: "Hello from an embedded binding"
```

What I like about this is that it is a lot less verbose and more direct (we don't need the extra spec and params field as well as the name field (what value does the name field have for an embedded binding). Though this is different from other places where we embed resources so we should discuss this first before implementing.

Reply:
Good idea. Would do in next PR.